calculator.rb

```diff
@@ -0,0 +1,96 @@
+#create a calculator program that asks for user input and runs file from command line
+
+#define global scope variables
+results = nil
+
+#define all the operations to the user that is available to compute calculations
+operations_list = {
+"Addition": "+",
+"Subtraction": "-",
+"Multiplication": "*",
+"Division": "/"
+}
+
+#define a list with the various names for the operations to use as user may use abbreviated operation terms
+variations_list = ["addition", "add", "+", "subtraction", "subtract", "-", "multiplication", "multiply", "*", "division", "divide", "/"]
+
+#method for checking if value is numeric
+def is_number? string
+    true if Float(string) rescue false
+end
+
+#method for calculating numbers
+def calculations (operator, num1, num2)
+    case operator
+    when "addition", "add", "+"
+        results = num1 + num2
+        puts "Your answer is: " + results.to_s
+    when "subtraction", "subtract", "-"
+        results = num1 - num2
+        puts "Your answer is: " + results.to_s
+    when "multiplication", "multiply", "*"
+        results = num1 * num2
+        puts "Your answer is: " + results.to_s
+    when "division", "divide", "/"
+        #check to see if user is trying to divide by 0
+        if num2 == 0
+            #if num2 is 0, then tell user, not possible. That's rude and exit program. 
+            puts "Looks like you're trying to divide by 0. You know that's mean. BYE BYE FELICIA!"
+            exit 
+            #else calculate 
+            results = num1 / num2
+            puts "Your answer is: " + results.to_s
+        end 
+    end 
+end 
+
+#welcome message and list of operations for user to pick from
+puts "We gon' do some math!"
+puts "Here is a list of types of math"
+operations_list.each do |key, value|
+  puts "#{key} (#{value})"
+end
```

Comment:
this is so fancy and it makes me happy. Maybe consider renaming `key` and `value` to `operation` and `symbol`?